src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/Interop/LdapPal.Linux.cs

```diff
@@ -123,7 +123,7 @@ internal static int BindToDirectory(ConnectionHandle ld, string who, string pass
             }
         }
 
-        internal static int StartTls(ConnectionHandle ldapHandle, ref int ServerReturnValue, ref IntPtr Message, IntPtr ServerControls, IntPtr ClientControls) => Interop.Ldap.ldap_start_tls(ldapHandle, ref ServerReturnValue, ref Message, ServerControls, ClientControls);
+        internal static int StartTls(ConnectionHandle ldapHandle, IntPtr serverControls, IntPtr clientControls) => Interop.Ldap.ldap_start_tls(ldapHandle, serverControls, clientControls);
```

Comment:
I know a little bit more now, so I think I shouldn't be modifying the signature of this method, but should just pass the relevant arguments from `LdapPal.StartTls()` to `Interop.Ldap.ldap_start_tls()`, and ignore the rest. Is that correct?